.github/workflows/shinyapps_deploy.yml

```diff
@@ -57,11 +57,14 @@ jobs:
           if [[ $GITHUB_REF_NAME == v*.*.* ]] || [[ $GITHUB_REF_NAME == main ]]; then
             echo Installing pypi version of schematic
             git clone --single-branch --branch main https://github.com/Sage-Bionetworks/schematic.git
+            cp schematic_config.yml schematic/config.yml
+            cd schematic
```

Comment:
`pip install` should not be affected the current working directory, while `poetry install` requires to be in the "schematic" folder.